popup.js

```diff
@@ -64,7 +64,7 @@ const modalData = [
     },
     image: './images/snap3.png',
     description:
-      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
     tech: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
     button: {
       link: 'https://example.com/',
```

Comment:
- [ ] Just as mentioned by the previous reviewer, the source link has a broken image tag here. Your whole code was commented, so I suggested you trace the icon or image link to get that fixed 👍 

# View:
![image](https://user-images.githubusercontent.com/50990925/216736145-f3dd18ad-9a28-434b-a249-ef3b475eab79.png)

# Figma view:
![image](https://user-images.githubusercontent.com/50990925/216736202-c36d6e02-ee0f-42e5-b4e2-a26ba8f5e40f.png)